src/service.rs

```diff
@@ -170,6 +175,7 @@ impl Service {
             stat_prefix: "ingress_http".to_string(),
             codec_type: 0,
             http_filters: vec![
+                http_filter.unwrap(),
```

Comment:
If you call `unwrap()` on it, it is no longer optional - you will want to create the vec outside the definition of `connection_manager`, optionally push the filter passed in, and push the filter we are creating here as well.